pkgs/games/xonotic/default.nix

```diff
@@ -1,7 +1,7 @@
 { lib, stdenv, fetchurl, fetchzip, makeWrapper, runCommandNoCC, makeDesktopItem
-, xonotic-data
```

Comment:
The intention of this was to make the data derivation overridable, but I'm not sure if that worked before your changes because I added a 'rec' below.